src/index.ts

```diff
@@ -1,19 +1,27 @@
-const {extract} = require('tar');
-const {execFileSync, execSync} = require('child_process');
-const path = require('path');
-const isImage = require('is-image');
-
-const unsupportedExtensions = new Set(['ai', 'emf', 'eps', 'gif', 'ico', 'psd', 'svg']);
+import {extract} from 'tar';
+import {execFileSync, execSync} from 'child_process';
+import path from 'path';
+import isImage from 'is-image';
+
+const unsupportedExtensions: Set<string> = new Set([
```

Comment:
```suggestion
const unsupportedExtensions = new Set([
```

This is already implicitly derived by TS, no need for such redundant type hints

Reply:
It's the same reason why you don't need to tell that `const foo: string = 'bar'` as TS already sees that `'bar'` is a string